src/test/java/teammates/test/cases/datatransfer/FeedbackSessionResultsBundleTest.java

```diff
@@ -0,0 +1,64 @@
+package teammates.test.cases.datatransfer;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.CourseRoster;
+import teammates.common.datatransfer.DataBundle;
+import teammates.common.datatransfer.FeedbackSessionResultsBundle;
+import teammates.common.datatransfer.attributes.FeedbackQuestionAttributes;
+import teammates.common.datatransfer.attributes.FeedbackResponseAttributes;
+import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
+import teammates.test.cases.BaseTestCase;
+
+/**
+ * SUT: {@link teammates.common.datatransfer.FeedbackSessionResultsBundle}.
+ */
+public class FeedbackSessionResultsBundleTest extends BaseTestCase {
+
+    @Test
+    public void testGetAllResponses() {
+        DataBundle responseBundle = loadDataBundle("/FeedbackSessionResultsBundleTest.json");
```

Comment:
Can use `typicalDataBundle.json`

Reply:
Actually `typicalDataBundle.json` has large number of responses from which I took only the once I needed. I made a lot of changes to this new file like faking responseId and making sure questionId of responses matches actual questions etc to avoid getting null pointer exceptions during construction of `FeedbackSessionResultsBundle`. So using `typicalDataBundle.json` would mean making changes to it (which would be really cumbersome) and it may break other tests using this json file.